packages/kbn-ftr-common-functional-ui-services/index.ts

```diff
@@ -22,3 +22,4 @@ export {
 } from './services/remote/network_profiles';
 export type { TimeoutOpt } from './types';
 export { TestSubjects } from './services/test_subjects';
+export { ToastsService } from './services/toasts';
```

Comment:
AFAICS we never import this, so this could be removed. Or is there a reason to keep it?